db/db.go

```diff
@@ -28,6 +28,7 @@ type DB interface {
 	UpsertFetchMeta(*models.FetchMeta) error
 
 	Get(string) (*models.CveDetail, error)
+	GetAll(limit, pageNum int) ([]models.CveDetail, int, error)
```

Comment:
The function name GetAll gives the impression that it will retrieve all CVEs.
```suggestion
	GetPage(limit, pageNum int) ([]models.CveDetail, int, error)
```

Reply:
Thank you for your suggestion to rename the GetAll function to GetPage. However, I've decided to stick with GetAll for a few reasons:

Consistency: Our codebase follows a naming convention where functions that retrieve data in a paginated manner are prefixed with GetAll. This helps in maintaining consistency across our code.

Clarity: The name GetAll clearly indicates the function's purpose is to retrieve all records, albeit in a paginated format. It aligns with the intention that the function is capable of fetching the entire dataset, controlled by pagination parameters.

Convention: In many frameworks and libraries, it's a common practice to use GetAll for functions that support retrieval of all items, with pagination often being an implicit or optional feature.